Task14/src/main/java/bsu/tat/triangle/TriangleDefine.java

```diff
@@ -0,0 +1,56 @@
+package bsu.tat.triangle;
+
+/**
+ * Class which define type of triangle
+ *
+ * @author Aleksey Guidnsky
+ */
+public class TriangleDefine {
+    static TriangleDefine td = new TriangleDefine();
+
+    /**
+     * Validate if type is equilateral, if not calls next method
+     *
+     * @param a is value of side a
+     * @param b is value of side b
+     * @param c is value of side c
+     */
+    public void defineType(double a, double b, double c) {
+        if ((a == b) & (b == c)) {
+            System.out.println(Messages.msg.EQUILATERAL);
+            return;
+        } else {
+            defineIsosceles(a, b, c);
+        }
+    }
+
+    /**
+     * Validate if type is isosceles, if not calls next method
+     *
+     * @param a is value of side a
+     * @param b is value of side b
+     * @param c is value of side c
+     */
+    public void defineIsosceles(double a, double b, double c) {
+        if ((a == b || (a == c) || (b == c))) {
+            System.out.println(Messages.msg.ISOSCELES);
```

Comment:
You won't be able to test - method must return some constant instead of printing to console